packages/webdriverio/src/scripts/newWindow.ts

```diff
@@ -11,5 +11,5 @@ export default function newWindow (
     windowName: string,
     windowFeatures: string
 ) {
-    window.open(url, windowName || 'new window', windowFeatures || '')
+    window.open(url, windowName || '', windowFeatures || '')
```

Comment:
Is assigning no window name the same as assigning a random one?

Reply:
@christian-bromann  yes, window.open(url,undefined,feature) and window.open(url,"",feature) both opens new windows each time which would you prefer